docs/api.md

```diff
@@ -822,3 +846,49 @@ $ curl -s "http://localhost:3100/loki/api/v1/series" --data-urlencode 'match={co
   ]
 }
 ```
+
+## Statistics
+
+Query endpoints such as `/api/prom/query`, `/loki/api/v1/query` and `/loki/api/v1/query_range` return a set of statistics about the query execution. Those statistics allow users to understand the amount of data processed and at which speed.
+
+The example belows show all possible statistics returned with their respective description.
+
+```json
+{
+  "status": "success",
+  "data": {
+    "resultType": "streams",
+    "result": [],
+    "stats": {
+     "ingester" : {
+        "compressedBytes": 0, // Total bytes of compressed chunks (blocks) processed by ingesters
+        "decompressedBytes": 0, // Total bytes decompressed and processed by ingesters
+        "decompressedLines": 0, // Total lines decompressed and processed by ingesters
+        "headChunkBytes": 0, // Total bytes read from ingesters head chunks
+        "headChunkLines": 0, // Total lines read from ingesters head chunks
+        "totalBatches": 0, // Total batches sent by ingesters
+        "totalChunksMatched": 0, // Total chunks matched by ingesters
+        "totalDuplicates": 0, // Total of duplicates found by ingesters
+        "totalLinesSent": 0, // Total lines sent by ingesters
+        "totalReached": 0 // Amount of ingesters reached.
+      },
+      "store": {
+        "compressedBytes": 0, // Total bytes of compressed chunks (blocks) processed by the store
+        "decompressedBytes": 0,  // Total bytes decompressed and processed by the store
+        "decompressedLines": 0, // Total lines decompressed and processed by the store
+        "chunksDownloadTime": 0, // Total time spent downloading chunks in seconds (float)
+        "totalChunksRef": 0, // Total chunks found in the index for the current query
+        "totalChunksDownloaded": 0, // Total of chunks downloaded
+        "totalDuplicates": 0 // Total of duplicates removed from replication
+      },
+      "summary": {
```

Comment:
I'm wondering if we should just document `summary`. I think others are interesting when troubleshooting but not really from a user perspective.